jobs/uaa/templates/config/uaa.yml.erb

```diff
@@ -504,9 +504,6 @@
       },
       'addShadowUserOnLogin' => p('uaa.ldap.add_shadow_user_on_login')
     }
-    unless p_opt('uaa.ldap.sslCertificate').nil?
-      ldap['ssl']['sslCertificate'] = p_opt('uaa.ldap.sslCertificate')
```

Comment:
Wouldn't this go beyond deprecation into removal territory?

Reply:
I'm >99% sure this property is never read out of `uaa.yml`. Removing it from here just clarifies that.